### ISSUE TITLE
build HTML docs from TS/JSDoc

### ISSUE THREAD

**Issue by turadg** [endojs/endo#1880]
## What is the Problem Being Solved?

Endo API is pretty vast but not browsable as documentation. docs.agoric.com points to https://github.com/endojs/endo/blob/HEAD/packages/ses/docs/guide.md

Eventually there should be a web site documenting Endo's API in its current state. This requires auto-generation from source docs.

First we need to be able to build such a site, which requires the types to resolve to a single stable configuration for the whole repo.

## Description of the Design

Basically like https://github.com/Agoric/agoric-sdk/pull/8343

Just building. Publishing will be separate work
